### DISCUSSION TITLE
Welcome to esp32-scoring-device Discussions!

### DISCUSSION THREAD

**Discussion by pietwauters** [pietwauters/esp32-scoring-device#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by cazanella** [comment]
Hi Piet,

I’m an electronics engineer from Brazil and I’ve been into fencing since I was a kid. I fenced from age 14 to 24, and about 3 years ago I got back into it as a veteran (I’m 59 now).

I’ve recently bought the domain **esgrima.ai** because I want to help grow the sport and make it more popular. I’m still figuring out exactly what to do with the site, but my initial idea is to share projects for fencing scoring boxes, weapon testers, DIY reels—basically anything that can help smaller clubs. I want to focus on cost-effective solutions, giving full credit and linking back to the original creators.

I’m also considering selling kits or pre-assembled products for those who aren't as tech-savvy, perhaps through partnerships to make them available worldwide.

I’ve been following you for a long time, and I was surprised to see today that you’re building something very similar with 'OpenPiste.' I’d love to bring more people in and build a community for exchanging ideas, projects, and improvements.

What do you think? If you’re interested, I’d love to chat more, refine the plan, and get the page live.

Best regards,

**Contacts:**
carloszanella@gmail.com
Phone: +55 41 98428-5494

**Reply by pietwauters** [reply]
Hi Carlos,
that is exactly my intention too. 
I'm working with a few friends to offer the main PCB's fully assembled, so even the less technical people can easily assemble the devices.
So yes, let's work together. I have been working intensively over the past months to improve the tester, scoring device and remote control. Besides that, I also have a few 3D printed tools to make the life of the armourors a bit easier. I you intend to build any of the devices, let me know and make sure to use the latest repositories.

Piet

**Comment by imammuslim833-ux** [comment]
Hi,
I found your GitHub profile and was impressed by your AI/ML experience and projects.
I’m looking for someone with strong English communication and technical interview experience for a remote collaboration opportunity related to AI/ML interviews.
Would you be interested in discussing further?

Thank you.

**Comment by cazanella** [comment]
Hi  Piet

Sorry for the delay in getting back to you, I've been traveling for work
for the past two weeks.

I'm going to create the website layout and add some useful links, videos,
the FIE website, and the projects with explanations on how they work,
photos, and then direct everything to their respective pages.

Some boards I ordered from China for your devices have arrived. I'm going
to assemble them just like the weapon tester, and I'll include that on the
site too.

This week, I'm going to look for someone who can develop the site so we can
keep it updated with new info later. I imagine it should be up and running
by June 1st.

If you know anyone else who might want to contribute with something, they
are more than welcome.

Best,

Carlos Zanella

Em qui., 7 de mai. de 2026 às 06:49, Piet Wauters ***@***.***>
escreveu:

> Hi Carlos,
> that is exactly my intention too.
> I'm working with a few friends to offer the main PCB's fully assembled, so
> even the less technical people can easily assemble the devices.
> So yes, let's work together. I have been working intensively over the past
> months to improve the tester, scoring device and remote control. Besides
> that, I also have a few 3D printed tools to make the life of the armourors
> a bit easier. I you intend to build any of the devices, let me know and
> make sure to use the latest repositories.
>
> Piet
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/pietwauters/esp32-scoring-device/discussions/1#discussioncomment-16839384>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A7MEZYZSOD6MCYJG6NGLUCD4ZRL2RAVCNFSM6AAAAACYTXQFJGVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTMOBTHEZTQNA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you commented.Message ID:
> <pietwauters/esp32-scoring-device/repo-discussions/1/comments/16839384@
> github.com>
>


-- 

[image: photo]

Carlos Alberto Zanella
Engenheiro e Esgrimista

+55 41 98428-5494 <+554198428-5494>  |  ***@***.***

[image: facebook] <https://www.facebook.com/carlosalberto.zanella.3>

[image: instagram] <https://www.instagram.com/carlos.alberto.zanella/>

[image: linkedin] <https://www.linkedin.com/in/carloszanella/>
Create your own email signature
<https://www.wisestamp.com/lp/promo/professional-email-signature?utm_source=promotion&utm_medium=signature&utm_campaign=create_your_own&srcid=>

[image: __tpx__]
